minor fix on typo

tuxerrante opened the pull request:
Explicit memory settings for new users. Little typo.

tuxerrante committed:
minor fix

Explicit memory settings for new users. Little typo.

rhuss commented on foundational/PredictableDemands/README.adoc:
```diff
@@ -10,7 +10,7 @@ To access the PersistentVolume used in this demo, let's mount a local directory
 [source, bash]
 ----
-minikube start --mount --mount-string="$(pwd)/logs:/example"
+minikube start --mount --mount-string="$(pwd)/logs:/example" --memory 2g
```
Particular reason why you do need the `--memory` option here ?

rhuss reviewed (commented):
Thanks ! Looks good but I have a question about the memory option that you have introduced

tuxerrante commented:
I've just added it to make it more explicit how to increase it when someone
is copy pasting fast the code.
In my case for knative I had to increase it to 8g.
Nothing special :)

Il Sab 20 Feb 2021, 17:18 Roland Huß <notifications@github.com> ha scritto:

> *@rhuss* commented on this pull request.
>
> Thanks ! Looks good but I have a question about the memory option that you
> have introduced
> ------------------------------
>
> In foundational/PredictableDemands/README.adoc
> <https://github.com/k8spatterns/examples/pull/20#discussion_r579676017>:
>
> > @@ -10,7 +10,7 @@ To access the PersistentVolume used in this demo, let's mount a local directory
>
>  [source, bash]
>  ----
> -minikube start --mount --mount-string="$(pwd)/logs:/example"
> +minikube start --mount --mount-string="$(pwd)/logs:/example" --memory 2g
>
> Particular reason why you do need the --memory option here ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/k8spatterns/examples/pull/20#pullrequestreview-594764986>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB72DNN524FDISS76CJ5MZDS77OEHANCNFSM4X4JQTBQ>
> .
>

snyk-bot committed:
fix: configuration/ImmutableConfiguration/init-container/Dockerfile-demo to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-DEBIAN8-CURL-358771
- https://snyk.io/vuln/SNYK-DEBIAN8-CURL-466507
- https://snyk.io/vuln/SNYK-DEBIAN8-SYSTEMD-305088
- https://snyk.io/vuln/SNYK-DEBIAN8-WGET-300469
- https://snyk.io/vuln/SNYK-DEBIAN8-WGET-342200

snyk-bot committed:
fix: configuration/ImmutableConfiguration/docker-volumes/Dockerfile-demo to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-DEBIAN8-CURL-358771
- https://snyk.io/vuln/SNYK-DEBIAN8-CURL-466507
- https://snyk.io/vuln/SNYK-DEBIAN8-SYSTEMD-305088
- https://snyk.io/vuln/SNYK-DEBIAN8-WGET-300469
- https://snyk.io/vuln/SNYK-DEBIAN8-WGET-342200

snyk-bot committed:
fix: configuration/ImmutableConfiguration/init-container/Dockerfile-demo to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-DEBIAN8-CURL-358771
- https://snyk.io/vuln/SNYK-DEBIAN8-CURL-466507
- https://snyk.io/vuln/SNYK-DEBIAN8-SYSTEMD-305088
- https://snyk.io/vuln/SNYK-DEBIAN8-WGET-300469
- https://snyk.io/vuln/SNYK-DEBIAN8-WGET-342200

tuxerrante committed:
Merge pull request #2 from tuxerrante/snyk-fix-cac59ccb98cf921d41015eab6db129e5

[Snyk] Security upgrade tomcat from 8.0 to 8.5.71

tuxerrante committed:
Merge branch 'master' into snyk-fix-0fbc36263f9dd26879d78ee8effa9c44

tuxerrante committed:
Merge pull request #8 from tuxerrante/snyk-fix-0fbc36263f9dd26879d78ee8effa9c44

[Snyk] Security upgrade tomcat from 8.0 to 8

tuxerrante committed:
Merge pull request #7 from tuxerrante/snyk-fix-818e774a9b88e99155c5b2f01e3d37cc

[Snyk] Security upgrade tomcat from 8.0 to 8.5.73

snyk-bot committed:
fix: structural/Ambassador/image/Dockerfile to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-DEBIAN11-CURL-1585138
- https://snyk.io/vuln/SNYK-DEBIAN11-CURL-1585138
- https://snyk.io/vuln/SNYK-DEBIAN11-CURL-1585138
- https://snyk.io/vuln/SNYK-DEBIAN11-CURL-1585138
- https://snyk.io/vuln/SNYK-DEBIAN11-PYTHON39-1290158

tuxerrante committed:
Merge pull request #9 from tuxerrante/snyk-fix-fa8905d538b6548b38ee3aff4a23fd89

[Snyk] Security upgrade node from latest to current-slim

snyk-bot committed:
fix: advanced/ImageBuilder/openshift/Dockerfile.multi-stage to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-ALPINE39-BZIP2-452847
- https://snyk.io/vuln/SNYK-ALPINE39-MUSL-458529
- https://snyk.io/vuln/SNYK-ALPINE39-MUSL-458529
- https://snyk.io/vuln/SNYK-ALPINE39-SQLITE-449671
- https://snyk.io/vuln/SNYK-ALPINE39-SQLITE-587441

tuxerrante committed:
Merge pull request #10 from tuxerrante/snyk-fix-6d20c043c11d96bbce9cf8ae005b89ac

[Snyk] Security upgrade openjdk from 8-alpine to 18-alpine

snyk-bot committed:
fix: structural/Adapter/image/Dockerfile to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-DEBIAN11-CURL-3065656
- https://snyk.io/vuln/SNYK-DEBIAN11-LIBTASN16-3061097
- https://snyk.io/vuln/SNYK-DEBIAN11-TIFF-3113871
- https://snyk.io/vuln/SNYK-DEBIAN11-TIFF-3113871
- https://snyk.io/vuln/SNYK-DEBIAN11-TIFF-3113871

tuxerrante committed:
Merge pull request #35 from tuxerrante/snyk-fix-3c830b4aea023e0bff887f5c755318d6

[Snyk] Security upgrade perl from latest to slim-threaded-buster

tuxerrante committed:
merge head (#36)

* fix typo can --> can't

* typo fix

* add indexed jobs

* Updated BatchJob and PeriodicJob patterns.

* Updated EnvVar examples for second edition

* some minor updates

* added sceletons for new chapters

Co-authored-by: Michael Hogan <michael.c.hogan2@boeing.com>
Co-authored-by: Roland Huß <roland@ro14nd.de>
Co-authored-by: Roland Huß <rhuss@redhat.com>

snyk-bot committed:
fix: configuration/ImmutableConfiguration/docker-volumes/Dockerfile-demo to reduce vulnerabilities (#37)

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-DEBIAN11-CURL-2936229
- https://snyk.io/vuln/SNYK-DEBIAN11-DPKG-2847942
- https://snyk.io/vuln/SNYK-DEBIAN11-LIBTASN16-3061097
- https://snyk.io/vuln/SNYK-DEBIAN11-OPENSSL-2807596
- https://snyk.io/vuln/SNYK-DEBIAN11-OPENSSL-2933518

snyk-bot committed:
fix: advanced/ImageBuilder/openshift/Dockerfile.multi-stage to reduce vulnerabilities (#38)

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-ALPINE315-LIBTASN1-3173576
- https://snyk.io/vuln/SNYK-ALPINE315-OPENSSL-2426331
- https://snyk.io/vuln/SNYK-ALPINE315-OPENSSL-2426331
- https://snyk.io/vuln/SNYK-ALPINE315-ZLIB-2434420
- https://snyk.io/vuln/SNYK-ALPINE315-ZLIB-2976173

snyk-bot committed:
fix: advanced/ImageBuilder/openshift/Dockerfile.multi-stage to reduce vulnerabilities (#39)

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-ALPINE315-LIBTASN1-3173576
- https://snyk.io/vuln/SNYK-ALPINE315-OPENSSL-2426331
- https://snyk.io/vuln/SNYK-ALPINE315-OPENSSL-2426331
- https://snyk.io/vuln/SNYK-ALPINE315-ZLIB-2434420
- https://snyk.io/vuln/SNYK-ALPINE315-ZLIB-2976173

snyk-bot committed:
fix: advanced/ImageBuilder/openshift/Dockerfile.multi-stage to reduce vulnerabilities (#40)

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-ALPINE315-LIBTASN1-3173576
- https://snyk.io/vuln/SNYK-ALPINE315-OPENSSL-2426331
- https://snyk.io/vuln/SNYK-ALPINE315-OPENSSL-2426331
- https://snyk.io/vuln/SNYK-ALPINE315-ZLIB-2434420
- https://snyk.io/vuln/SNYK-ALPINE315-ZLIB-2976173

snyk-bot committed:
fix: configuration/ImmutableConfiguration/docker-volumes/Dockerfile-demo to reduce vulnerabilities (#41)

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-UBUNTU2204-CURL-3070972
- https://snyk.io/vuln/SNYK-UBUNTU2204-CURL-3180245
- https://snyk.io/vuln/SNYK-UBUNTU2204-CURL-3180260
- https://snyk.io/vuln/SNYK-UBUNTU2204-OPENSSL-3092568
- https://snyk.io/vuln/SNYK-UBUNTU2204-OPENSSL-3092591

rhuss commented:
Oops, the PR seems to be messed up quite a bit. Could you briefly summarize the typo that you have fixed, so I just would add it manually ? thx!

tuxerrante commented:
I'd say it was just a fix from "the" to "to" at line 184
https://github.com/k8spatterns/examples/pull/20/commits/cad4b5fabe5d73f22816c63dcabfc04e77306b31

And the "--memory" argument mentioned above
thanks

snyk-bot committed:
fix: structural/Adapter/image/Dockerfile to reduce vulnerabilities (#42)

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-DEBIAN10-GLIBC-559488
- https://snyk.io/vuln/SNYK-DEBIAN10-GLIBC-559488
- https://snyk.io/vuln/SNYK-DEBIAN10-GLIBC-559488
- https://snyk.io/vuln/SNYK-DEBIAN10-GLIBC-559488
- https://snyk.io/vuln/SNYK-DEBIAN10-LIBTASN16-3061094

rhuss commented:
Thanks ! I added the change manually, so closing this PR
